packages/eth2.0-state-transition/src/epoch/registryUpdates.ts

```diff
@@ -20,9 +20,9 @@ export function processRegistryUpdates(config: IBeaconConfig, state: BeaconState
   const currentEpoch = getCurrentEpoch(config, state);
   // Process activation eligibility and ejections
   const ejectionBalance = config.params.EJECTION_BALANCE;
-  state.validators.forEach((validator, index) => {
+  state.validators.forEach((validator, index, validators) => {
     if (isEligibleForActivationQueue(config, validator)) {
-      validator.activationEligibilityEpoch = currentEpoch + 1;
```

Comment:
why?

Reply:
stupid webstorm got confused and auto corrected I guess. I think it said that primitive value would be lost for some reason. I will revert tommorow